Makefile

```diff
@@ -17,6 +17,8 @@ endif
 all: build
 
 build:
+	# Permissions workaround for global `npm install`
```

Comment:
Just added as small code comment here about why that chown was needed. This may help someone in future reading this Makefile.